fixtures/coverall/tests/bindings/test_coverall.py

```diff
@@ -84,5 +91,65 @@ def test_errors(self):
         with self.assertRaisesRegex(InternalError, "expected panic: oh no"):
             coveralls.panic("expected panic: oh no")
 
+    def test_arcs(self):
+        coveralls = Coveralls("test_arcs")
+        self.assertEqual(get_num_alive(), 1)
+        self.assertEqual(coveralls.strong_count(), 2)
```

Comment:
To check my understanding: the strong count is 2 because 1 reference is held by the foreign-language code, and one reference is held by the Rust code for the duration of the call?

Reply:
yes, in this specific example, 1 is in `coveralls._handle` and the other is a new Arc created for `self: Arc<Self>`